.changeset/rotten-tomatoes-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-docs/gatsby-transformer-raml': minor
```

Comment:
Since it is more a bugfix than a new feature, I would say it's more a patch than a minor change

Reply:
I agree 👍